src/viam/examples/CMakeLists.txt

```diff
@@ -13,6 +13,5 @@
 # limitations under the License.
 
 add_subdirectory(dial)
```

Comment:
I took the opportunity to remove the `dial` example. I don't think it served much purpose anymore and it had some odd effects on the build system.

Reply:
You say `dial` in your comment but you removed `echo`. Typo?

Reply:
Yes, thanks. I removed `echo`.